base_wamas_ubl/tests/test_base_wamas_ubl.py

```diff
@@ -75,6 +77,31 @@ def setUpClass(cls):
                 and cls.partner_2.child_ids[0].email
                 or "",
             },
+            "MAPPING_UNITCODE_WAMAS_TO_UBL": {
+                "unitCode": MappingDict(
+                    {
+                        "BOT": "XBQ",  # plastic bottle
+                        "BOUT": "C62",  # Unit
+                        "BOITE": "XBX",  # box
+                        "LITRE": "LTR",  # litre
+                        "PET": "XBO",  # glass bottle
+                        "TETRA": "X4A",  # tetra pack, changed 'X4B' to 'X4A'for testing
```

Comment:
Hi reviewer, [the default of 'TETRA'](https://github.com/OCA/edi/blob/644353cf7b25b9bfd91f36368c4f1c2b47ea9289/base_wamas_ubl/lib/wamas/const.py#L190) is X4B, so i just want to change to X4A to test

To make sure that it override the default no